### PR TITLE
Doc fixes

### PR DESCRIPTION
**Pull request by ktorpi**

### CODE REVIEW

File: 03.api.md [hunk 1]
```diff
@@ -303,147 +312,989 @@ jerry_exec_snapshot (const void *snapshot_p,
...
+jerry_api_value_t
+jerry_api_create_void_value (void);
+```
+
+- return value - a jerry_api_value_t with type JERRY_API_DATA_TYPE_VOID
+
+**Example**
+
+```c
+  jerry_init(JERRY_FLAG_EMPTY);
```
Comment (zherczeg) [inline 03.api.md]: Space missing

File: 03.api.md [hunk 2]
```diff
@@ -303,147 +312,989 @@ jerry_exec_snapshot (const void *snapshot_p,
...
+  jerry_api_value_t void_value = jerry_api_create_void_value();
```
Comment (zherczeg) [inline 03.api.md]: Ditto.

File: 03.api.md [hunk 3]
```diff
@@ -303,147 +312,989 @@ jerry_exec_snapshot (const void *snapshot_p,
...
+jerry_api_create_null_value (void);
+```
+
+- return value - a jerry_api_value_t with type JERRY_API_DATA_TYPE_NULL
+
+**Example**
+
+```c
+{
+  jerry_init(JERRY_FLAG_EMPTY);
```
Comment (zherczeg) [inline 03.api.md]: Please fix all of them.

File: 03.api.md [hunk 4]
```diff
@@ -1014,37 +1882,39 @@ jerry_api_get_object_field_value_sz (jerry_api_object_t *object_p,
...
-                                                       (jerry_api_size_t)strlen((char*)field_name),
-                                                       &retrieved_field_value);
-
-  // usage of the retrieved field...
-
+  bool is_get_ok = jerry_api_get_object_field_value_sz (object_p,
+                                                        field_name,
+                                                        lit_zt_utf8_string_size (field_name),
+                                                        &retrieved_field_value);
+  if (is_get_ok) {
```
Comment (zherczeg) [inline 03.api.md]: The { is on the bad side

**Comment by zherczeg** [comment]
LGTM

**Comment by LaszloLango** [comment]
@ktorpi, please reabse the PR

**Comment by ktorpi** [comment]
@LaszloLango, I've rebased it

**Comment by LaszloLango** [comment]
@ktorpi, thanks. LGTM

**Pull request merged**